app/main.py

```diff
@@ -1,3 +1,46 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: [float, int]) -> any:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {str(self.km)} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: [int, float, Distance]) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: [int, float, Distance]) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
+        return self
+
+    def __mul__(self, other: [int, float, Distance]) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: [int, float, Distance]) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: [int, float]) -> bool:
+        return self.km < other
+
+    def __gt__(self, other: [int, float]) -> bool:
+        return self.km > other
+
+    def __eq__(self, other: [int, float]) -> bool:
+        return self.km == other
+
+    def __le__(self, other: [int, float]) -> bool:
+        return self.km <= other
+
+    def __ge__(self, other: [int, float]) -> bool:
+        return self.km >= other
```

Comment:
this methods should work with Distnace instance, so add a check for instance class